src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -71,6 +71,12 @@ class SetupRecoveryMethod extends Component {
         }
     }
 
+    componentDidUpdate(prevProps, prevState) {
+        if (prevState.success !== this.state.success) {
```

Comment:
@Patrick1904 any reason not to do it in `handleNext`?  Seems more straightforward.

Reply:
Good idea. Fixed.